### ISSUE TITLE
Get a compilation error "CS1009: Unrecognized escape sequence" when using verbatim strings

### ISSUE THREAD

**Issue by davkean** [dotnet/BenchmarkDotNet#516]
``` C# 

using BenchmarkDotNet.Attributes;
using BenchmarkDotNet.Attributes.Jobs;
using BenchmarkDotNet.Configs;
using BenchmarkDotNet.Jobs;
using System;
using System.Collections.Generic;
using System.IO;
using System.Linq;
using System.Runtime.CompilerServices;
using System.Text;
using System.Threading.Tasks;

[LegacyJitX64Job]
public class IndexOfAny
{
    private const string File = @"Folder\Bar\Foo\Bar\FooBar.cs";

    [Params(File)]
    public string Input { get; set; }

    [Benchmark(Description = "Hand written loop over array (IndexOfAny)")]
    public void IndexOfAny_HandWrittenLoop()
    {
        for (int i = 0; i < 100000; i++)
        {
            ContainsHelper.IndexOfAny_HandWrittenLoop(Input, s_charsToEscape);
        }
    }

internal class ContainsHelper
{
    [MethodImpl(MethodImplOptions.NoInlining)]
    public static int IndexOfAny_HandWrittenLoop_HardcodedChars(string value)
    {
        for (int i = 0; i < value.Length; i++)
        {
            switch (value[i])
            {
                case '%':
                case '*':
                case '?':
                case '@':
                case '$':
                case '(':
                case ')':
                case ';':
                case '\'':
                    return i;
            }
        }

        return -1;
    }
}

 class Program
{
    static void Main(string[] args)
    {
        BenchmarkDotNet.Running.BenchmarkRunner.Run<IndexOfAny>();
    }
}

```

```
// **************************
// Benchmark: IndexOfAny.'Hand written loop over hardcoded chars (ContainsAny)': LegacyJitX64(Jit=LegacyJit, Platform=X64, Runtime=Clr) [Input=Folder\Bar\Foo\Bar\FooBar.cs]
// *** Generate ***
// Result = Success
// BinariesDirectoryPath = C:\Users\davkean\Source\Repos\PathHelpers\PathHelpers\bin\Release

// *** Build ***
BuildScript: C:\Users\davkean\Source\Repos\PathHelpers\PathHelpers\bin\Release\BDN.Generated.bat
CS1009: Unrecognized escape sequenceCS1009: Unrecognized escape sequenceCS1009: Unrecognized escape sequenceCS1009: Unrecognized escape sequence// Result = Failure
```

**Comment by davkean** [comment]
Hmm, I can't get this to work even if I change the verbatim string to escape sequence.

**Comment by AndreyAkinshin** [comment]
Hey @davkean, thanks for the report. The problem is: we use params values as parts of generated filenames (without escaping, unfortunately). Will fix it soon. For now, I can suggest the following workaround:
```cs
private string[] inputNames = new string[] { @"Folder\Bar\Foo\Bar\FooBar.cs" };

[Params(0)]
public int InputIndex;

// Benchmark
var input = inputNames[InputIndex];
```

**Comment by davkean** [comment]
Thanks for the quick reply and workaround!

**Comment by AndreyAkinshin** [comment]
@davkean, I added some escaping magic, the fix will be available in `v0.10.10`.

**Comment by davkean** [comment]
👍
